src/main/scala/tech/sourced/gemini/cmd/HashSparkApp.scala

```diff
@@ -55,7 +56,6 @@ object HashSparkApp extends App with Logging {
     opt[Int]("features-extractor-port")
       .action((x, c) => c.copy(fePort = x))
       .text("port is features-extractor server port")
-    opt[Unit]('v', "verbose")
```

Comment:
why did you remove reading verbose flag? And I see below you still use it.

Reply:
It was removed in aea49edbbfe980aee8d41d326d209ff131aec9df 

If defined twice as before, it would not be updated in Configuration and will aways result in `False` and thus verbose mode was never called. This tricky behavior costed few hours of debugging.

Reply:
Wow! Thanks for the explanation! Good catch!

Reply:
Np! Was quite confused with this behaviour as well.